ci-operator/step-registry/ingress-operator/ai-analyze/ingress-operator-ai-analyze-commands.sh

```
@@ -0,0 +1,57 @@
+    [[ $- == *x* ]] && WAS_TRACING=true || WAS_TRACING=false
+    set +x
+
+    NOW=$(date +%s)
+    IAT=$((NOW - 60))
+    EXP=$((NOW + 600))
+    HEADER=$(echo -n '{"alg":"RS256","typ":"JWT"}' | base64 | tr -d '=' | tr '/+' '_-' | tr -d '\n')
+    PAYLOAD=$(echo -n "{\"iat\":${IAT},\"exp\":${EXP},\"iss\":\"${APP_ID}\"}" | base64 | tr -d '=' | tr '/+' '_-' | tr -d '\n')
+    SIGNATURE=$(echo -n "${HEADER}.${PAYLOAD}" | openssl dgst -sha256 -sign "${PRIVATE_KEY}" | base64 | tr -d '=' | tr '/+' '_-' | tr -d '\n')
+    JWT="${HEADER}.${PAYLOAD}.${SIGNATURE}"
+
+    export GITHUB_TOKEN=$(curl -sf --connect-timeout 10 --max-time 30 --retry 3 --retry-delay 5 \
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)